frontend/src/constants/accessToken.ts

```diff
@@ -1,5 +1,8 @@
 import Cookies from 'js-cookie';
+import { uniqAccessToken } from 'api/authConstants';
 
-const accessToken = Cookies.get('accessToken');
+const token = Cookies.get(uniqAccessToken);
+let accessToken = '';
+token ? (accessToken = JSON.parse(token)) : (accessToken = '');
```

Comment:
Wrong definition of logic with assigning variable value. Rewrite it please with something more clear and readable.

Reply:
Created universal function for get cookies